app/models/course/assessment/answer/scribing_scribble.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+class Course::Assessment::Answer::ScribingScribble < ActiveRecord::Base
+  belongs_to :answer, class_name: Course::Assessment::Answer::Scribing.name, inverse_of: :scribbles
+  belongs_to :user, inverse_of: nil, foreign_key: true
```

Comment:
As far as I understand, in this case updater will always be the same as creator ?